changelog.d/10563.misc

```diff
@@ -0,0 +1 @@
+Update contributing.md to warn against rebasing an open PR.
```

Comment:
Maybe rename this to a `.doc`?

Reply:
given it's developer rather than admin doc, I don't think it matters too much :man_shrugging: